gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GcsJavaClientImpl.java

```diff
@@ -1,17 +1,25 @@
 package com.google.cloud.hadoop.gcsio;
```

Comment:
Please resolve conflicts before merging in `master` branch.

Reply:
Done.